saleor/graphql/order/mutations/orders.py

```diff
@@ -547,6 +547,57 @@ def perform_mutation(cls, _root, info, **data):
         return RequestInvoice()
 
 
+class CreateInvoiceInput(graphene.InputObjectType):
+    number = graphene.String(required=True, description="Invoice number")
+    url = graphene.String(required=True, description="URL of an invoice to download.")
+
+
+class CreateInvoice(ModelMutation):
+    class Arguments:
+        order_id = graphene.ID(
+            required=True, description="ID of the order related to invoice."
+        )
+        input = CreateInvoiceInput(
+            required=True, description="Fields required when creating an invoice."
+        )
+
+    class Meta:
+        description = "Creates an invoice."
+        model = models.Invoice
+        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        error_type_class = InvoiceError
+        error_type_field = "invoice_errors"
+
+    @classmethod
+    def clean_input(cls, info, instance, data):
+        cleaning_map = {
+            "url": InvoiceErrorCode.EMPTY_URL,
+            "number": InvoiceErrorCode.EMPTY_NUMBER,
+        }
+        for field, error_code in cleaning_map.items():
+            if data["input"][field] == "":
```

Comment:
ValidationError gives you a possibility to pass errors for more that one field. You can easily achieve it here.
Something like this:
```
    @classmethod
     def clean_input(cls, info, instance, data):
         input = data['input']
         validation_errors = {}
         url, number = input['url'], input['number']
         if not url:
             validation_errors['url'] = ValidationError(
                             f"{field} cannot be empty.", code=InvoiceErrorCode.EMPTY_URL,
                         )
         if not number:
             validation_errors['number'] = ValidationError(
                             f"{field} cannot be empty.", code=InvoiceErrorCode.EMPTY_NUMBER,
                         )
         if validation_errors:
            raise ValidationError(validation_errors)
```